pages/common/nnn.md

```diff
@@ -0,0 +1,36 @@
+# nnn
+
+> Interactive terminal file manager and fisk analyzer.
```

Comment:
```suggestion
> Interactive terminal file manager and disk usage analyzer.
```
Whoops, a typo (also, I forgot the important word "usage", since it's not a _disk analyzer_).